tests/integration/helpers.bash

```diff
@@ -19,6 +19,9 @@ HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 # CRIU PATH
 CRIU="/usr/local/sbin/criu"
 
+# CONFIG_MEMCG_KMEM support
+KMEM="/sys/fs/cgroup/memory/memory.kmem.limit_in_bytes"
```

Comment:
To be clear, current implementation in `cgroups.bats` is not right, see my comments in https://github.com/opencontainers/runc/pull/967#issuecomment-238060361